src/status_im/contexts/wallet/send/select_collectible_amount/view.cljs

```diff
@@ -0,0 +1,67 @@
+(ns status-im.contexts.wallet.send.select-collectible-amount.view
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [status-im.contexts.wallet.collectible.utils :as utils]
+    [status-im.contexts.wallet.common.account-switcher.view :as account-switcher]
+    [status-im.contexts.wallet.send.select-collectible-amount.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn view
+  []
+  (let [on-close              (rn/use-callback #(rf/dispatch [:navigate-back]))
+        [value set-value]     (rn/use-state 1)
+        inc-value             (rn/use-callback (fn [] (set-value (inc value)))
+                                               [value])
+        dec-value             (rn/use-callback (fn [] (set-value (dec value)))
+                                               [value])
+        add-digit             (rn/use-callback (fn [digit]
+                                                 (set-value (+ (js/parseInt digit)
+                                                               (* value 10))))
+                                               [value])
+        delete-digit          (rn/use-callback (fn []
+                                                 (set-value (Math/floor (/ value 10))))
+                                               [value])
+
+        send-transaction-data (rf/sub [:wallet/wallet-send])
+        collectible           (:collectible send-transaction-data)
+        balance               (utils/collectibles-count collectible)
+        preview-uri           (get-in collectible [:preview-url :uri])
+        incorrect-value?      (or (< value 1) (> value balance))]
+    [rn/safe-area-view
+     [account-switcher/view
+      {:icon-name     :i/arrow-left
+       :on-press      on-close
+       :switcher-type :select-account}]
+     [quo/expanded-collectible
+      {:image-src       preview-uri
+       :square?         true
+       :container-style style/collectible-container}]
+     [quo/network-tags
+      {:title           (str (i18n/label :t/max) ": " balance)
```

Comment:
Consider passing `balance` as a parameter to the i18n label. It will be easier to translate the string for RTL languages then.